pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -86,6 +86,8 @@ const (
 	CheckNodeMemoryPressurePred = "CheckNodeMemoryPressure"
 	// CheckNodeDiskPressurePred defines the name of predicate CheckNodeDiskPressure.
 	CheckNodeDiskPressurePred = "CheckNodeDiskPressure"
+	// CheckNodePIDPressurePred defines the name of predicate CheckNodePIDPressure.
+	CheckNodePIDPressurePred = "CheckNodePIDPressure"
```

Comment:
If we keep this as a predicates, we might want to add it to the predicatesOrdering list, see [predicates.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithm/predicates/predicates.go#L130)

Reply:
cc @bsalamat @resouer

Reply:
ok

Reply:
done